layersvt/CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ if (NOT APPLE)
 else()
     set(TARGET_NAMES
         VkLayer_api_dump
+	VkLayer_device_simulation
```

Comment:
Might want to fix this indent for neatness.

Reply:
Too quick on the trigger finger. Will fix right on master.